.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "files.autoSave": "off",
+  "editor.formatOnSave": false
+}
```

Comment:
Do you mind removing this?  I prefer to have formatting on save enabled.  And it looks like we're not checking prettier in CI, which we should add as well...

Reply:
I'll leave the checking-Prettier-in-CI part to you :)